syntax = "proto2";

package assets;

message Road {
  required uint32 id = 1;
  optional uint32 geojson_id = 2;
  optional string road_code = 3;
  optional string road_name = 4;
  optional string link_code = 5;
  optional string link_name = 6;
  optional float link_length = 7;
  optional string surface_type = 8;
  optional string surface_condition = 9;
  optional string road_type = 10;
}

message Roads {
  repeated Road roads = 1;
}
